dawn/webpack.config.babel.js

```diff
@@ -24,6 +24,7 @@ const plugins = [
   new webpack.DefinePlugin({
     VERSION: JSON.stringify(version),
   }),
+  new webpack.IgnorePlugin(/vertx/),
```

Comment:
Specifically I get an issue similar to https://github.com/redux-saga/redux-saga/issues/197. I'll debug on a separate branch.